stl/inc/bitset

```diff
@@ -20,6 +20,7 @@ _STL_DISABLE_CLANG_WARNINGS
 
 #pragma warning(disable : 6294) // Ill-defined for-loop:
                                 // initial condition does not satisfy test.  Loop body not executed.
+#pragma warning(disable : 4127) // conditional expression is constant

```

Comment:
I think this suppression should be in sorted order (i.e. before 6294). This warning is low-value enough that I don't think we need to scope it to `all()` (although we do that elsewhere).